src/pybind/mgr/dashboard/controllers/pool.py

```diff
@@ -15,7 +15,7 @@ class Pool(RESTController):
     @classmethod
     def _serialize_pool(cls, pool, attrs):
         if not attrs or not isinstance(attrs, list):
-            attrs = pool.keys()
+            attrs = list(pool.keys())
```

Comment:
So, `pool.keys()` returns an iterator in Python 3. But `attrs` is used only once in line 23. I'd think, this line should actually be OK, right?

Reply:
right, good catch!